src/contracts/MultiSigWallet.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: MIT
+// MultiSigWallet v0.1
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+contract MultiSigWallet is Ownable {
+    using SafeERC20 for IERC20;
+
+    address[] public _signers;
+
+    mapping(address => bool) public _isSigner;
+    mapping(address => bool) public _signed;
+
+    function appendSigner(address[] memory addresses_) public onlyOwner {
+        for (uint256 i = 0; i < addresses_.length; i++) {
+            _signers.push(addresses_[i]);
+            _isSigner[addresses_[i]] = true;
+        }
+    }
+
+    function removeSigner(address removeAddr_) public onlyOwner {
+        bool isRemoved = false;
+        for (uint256 i = 0; i < _signers.length; i++) {
+            if (removeAddr_ == _signers[i]) {
+                _signers[i] = _signers[_signers.length - 1];
+                _isSigner[removeAddr_] = false;
+                isRemoved = true;
+                break;
+            }
+        }
+        if (isRemoved) {
+            _signers.pop();
+        }
+    }
+
+    function isSignApproved() public view returns (bool) {
+        bool isApproved = true;
+        for (uint256 i = 0; i < _signers.length; i++) {
+            if (!_signed[_signers[i]]) {
+                isApproved = false;
+                break;
+            }
+        }
+        return isApproved;
+    }
+
+    function Sign() public {
```

Comment:
Is there any specific reasons you have that you made that a signer can sign, not multiple signers can sign concurrently?
Since this is the multisig contract, do we need to make the sign function that handles multiple addresses?
Just asking my opinion :)

Reply:
One of distribution addresses, is MultiSig Wallet.
So this contract is made. 
But after make, I heard you will use your own external multisig wallet.
In that case, this contract is not used.

Reply:
Got it. Thanks!